Added additional information about ANALYZE TABLE in TiDB (#970)
pingcap/docs: sql/statistics.md

### Manual collection

> You can run the `ANALYZE` statement to collect statistics. Note that `ANALYZE TABLE` in TiDB takes considerably longer than in MySQL/InnoDB. In InnoDB, only a small number of pages are sampled, while in TiDB a comprehensive set of statistics is completely rebuilt. Many scripts written for MySQL execute `ANALYZE TABLE` expecting that it will be a short-lived operation; those scripts should be modified not to make that same assumption about TiDB.

Reviewer: ```suggestion
You can run the `ANALYZE` statement to collect statistics. Note that `ANALYZE TABLE` in TiDB takes considerably longer than in MySQL/InnoDB. In InnoDB, only a small number of pages are sampled, while in TiDB a comprehensive set of statistics is completely rebuilt. Scripts that were written for MySQL may naively expect `ANALYZE TABLE` will be a short-lived operation.
```
